pkg/querier/querier.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2020 Antrea Authors
```

Comment:
Not the interface. I feel we should move implementation out of pkg monitor. I think we should put controller related query funcs to be under controller/, like pkg/controller/querier, and agent query funcs to be under agent/, like pkg/agent/querier.

controllerMonitor, agentMonitor, and controller/agent API implementations (inc. CLI commands handlers) can call these funcs. Only CRD struct creation, update, etc. should be put in the monitor/ pkg.

@tnqn: what are your opinions.

Reply:
Sounds good to me. I thought almost the same but didn't divide queriers to controller and agent because they have some shared interface/methods, where should the common codes be put you would expect?

Reply:
I think controller and agent share only three funcs: GetSelfPod, GetSelfNode, and GetVersion. If we want we can make three util funcs for them. Other funcs are quite specific to controller/agent.